### ISSUE TITLE
Logfile anonymizer

### ISSUE THREAD

**Issue by briantrice** [zero-motorcycle-community/zero-log-parser#2]
We need to be able to use logfiles for testing. The logfiles contain identifiable information which makes distribution sensitive.

Information in a logfile:
- VIN in the filename (easily scrubbed, just need to choose a method for it).
- VIN in the logfile (requires writing a new/blank value to relevant entries).
- Battery serial number in the logfile (requires writing a new/blank value to relevant entries).

**Comment by cecil-t** [comment]
A VIN consisting of 17 one's (11111111111111111) is technically a valid VIN by the algorithm, so that would be my recommendation to scrub it to if you write a routine.

However, I wouldn't consider a VIN an identifiable piece of information in the traditional sense of privacy / personal information security, there is nothing that ties a VIN to a person or registration other than state databases.  E.g. Carfax publishes VIN information publicly all the time, including previous registration city and state.

**Comment by keithxemi** [comment]
I agree with cecil-t's comment that VIN's are public if they are not paired with the identity of an individual. It may be impossible to anonymize test files, a hidden VIN could be somewhere in the log file, no way to know. In one case (2017 model) the VIN is not in the usual location, but is readable. Also, the VIN in the filename ensures a unique name, very important. I think we should include the test logs in the repo and encourage the collection of more.
